sdk/iothub/src/service/mod.rs

```diff
@@ -307,4 +457,60 @@ mod tests {
         let _ = ServiceClient::from_connection_string("HostName=cool-iot-hub.azure-devices.net;SharedAccessKey=YSB2ZXJ5IHNlY3VyZSBrZXkgaXMgaW1wb3J0YW50Cg==", 3600).is_err();
         Ok(())
     }
+
+    #[test]
+    fn update_module_twin_should_create_builder() -> Result<(), Box<dyn std::error::Error>> {
+        use crate::service::ServiceClient;
+        let connection_string = "HostName=cool-iot-hub.azure-devices.net;SharedAccessKeyName=iothubowner;SharedAccessKey=YSB2ZXJ5IHNlY3VyZSBrZXkgaXMgaW1wb3J0YW50Cg==";
```

Comment:
We should probably remove the hardcoded connection string and replace it with environment variable.

Reply:
I assume that would be needed for end-to-end tests if we were to add them. These currently are not end-to-end tests (yet). Is this something that is going to be required in the future? I can probably spend some time at a certain point on creating some end-to-end tests.